openssl-dynamic/src/main/c/sslcontext.c

```diff
@@ -348,6 +348,9 @@ TCN_IMPLEMENT_CALL(jlong, SSLContext, make)(TCN_STDARGS, jint protocol, jint mod
 TCN_IMPLEMENT_CALL(jint, SSLContext, free)(TCN_STDARGS, jlong ctx)
```

Comment:
did you consider the suggestion in https://github.com/netty/netty/pull/6254#discussion_r121768301 (change return type to jlong and return NULL)?